app/search/styles.scss

```diff
@@ -86,3 +94,77 @@
     display: flex;
     justify-content: space-between;
 }
+
+.search-help-tutorial-1,
+.search-help-tutorial-2,
+.search-help-tutorial-3 {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    background-color: $osf-dark-blue-navbar;
+    color: $color-text-white;
+    width: 400px;
+    margin-left: -0.2rem;
+
+    h2 {
+        font-weight: 400;
+        height: 22px;
+        margin-top: 0.5rem;
+        margin-left: 0.2rem;
+    }
+
+    aside > p {
+        height: 50px;
+        margin-top: 1rem;
+        padding-top: 0.2rem;
+        padding-bottom: 3rem;
+    }
+
+    p {
+        margin-left: 0.2rem;
+    }
+
+    button {
+        width: 65px;
+        color: $color-text-white;
+        border-radius: 2px;
+        overflow: visible;
+    }
+}
+
+.search-help-tutorial-2 {
+    margin-bottom: 1.1rem;
+}
+
+.search-help-tutorial-inner {
+    background-color: $osf-dark-blue-navbar;
+    color: $color-text-white;
+    width: 400px;
+    height: 158px;
+}
```

Comment:
This can be removed as it's not being applied to the tooltip

Reply:
Excellent, these are now removed.